doc/languages-frameworks/python.section.md

```diff
@@ -344,6 +344,16 @@ applied to the reference:
 youtube-dl = with python3Packages; toPythonApplication youtube-dl;
 ```
 
+Since the advent of `pkgs/by-name` hierarchy, there is an extra alternative:
+instead of writing the code above in `all-packages.nix`, the corresponding code
```

Comment:
Given the agreement that this is the preferred way to declare these types of packages going forward, can you reword this to fit in the previous paragraph (instead of "A reference shall be created...", reword to fit this text there instead)?

And then we can ellaborate on:
- Existing packages and why they must remain in `all-packages.nix`. I'd go with a note first (`:::{.note}`), but if the text becomes too large, can be a subsection.
- What to do in cases we need to override the package

Reply:
1. Unfortunately the override can be a bit complicated, since the "library" is accessed behind `python3Packages`. But it happens in both scenarios, so nothing was lost.
2. I am sprint-removing the ones in top-level.

That being said, I believe I will need to rewrite this whole text, showing the new vs the legacy method or something similar.